nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java

```diff
@@ -391,6 +382,14 @@ public void handleEvent(RuntimeServiceEvent event) {
                         for (String id : workQueueConfig.getQueueIds()) {
                             deactivateQueue(workQueueConfig.get(id));
                         }
+                        try {
+                            if (!shutdown(1, TimeUnit.MINUTES)) {
+                                log.error("Some processors are still active");
```

Comment:
later we should remove the ambiguous "processor" term which is not used in Nuxeo (only one occurence in workmanager)